src/app/[locale]/products/_components/categories/index.tsx

```
@@ -0,0 +1,67 @@
+import { useCategories } from "../../_hooks/use-categories";
+import ResetButton from "../common/reset-button";
+import CategoryItem from "./components/category-item";
+
+export default function CategoriesFilters() {
+    // Next.js navigation hooks
+    const router = useRouter();
+    const pathname = usePathname();
+    const searchParams = useSearchParams();
+
+    // Fetch categories using custom hook
+    const { data, isLoading, isError } = useCategories();
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)